Sources/VLCSettingsTableViewCell.m

```diff
@@ -0,0 +1,73 @@
+/*****************************************************************************
+ * VLCSettingsTableViewCell.m
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Mike JS. Choi <mkchoi212 # icloud.com>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+#import "VLCSettingsTableViewCell.h"
+#import "IASKSettingsReader.h"
+#import "IASKSwitch.h"
+#import "VLC_iOS-Swift.h"
+
+@implementation VLCSettingsTableViewCell
+
++ (VLCSettingsTableViewCell *)cellWithIdentifier:(NSString *)identifier target:(id)target {
+    NSArray *nibArray = [[NSBundle mainBundle] loadNibNamed:@"VLCSettingsTableViewCell" owner:nil options:nil];
+    VLCSettingsTableViewCell *cell = (VLCSettingsTableViewCell *)[nibArray lastObject];
+
+    if ([identifier isEqualToString:kIASKPSToggleSwitchSpecifier]) {
+        cell.selectionStyle = UITableViewCellSelectionStyleNone;
+        cell.accessoryView = [[IASKSwitch alloc] initWithFrame:CGRectZero];
+        [(IASKSwitch *)(cell.accessoryView) addTarget:target action:NSSelectorFromString(@"toggledValue:") forControlEvents:UIControlEventValueChanged];
```

Comment:
could we have the entire cell be a touch target and not the switch itself?

Reply:
Should the selection style still be .none or changed to .default?